shapely/coords.py

```diff
@@ -46,8 +46,13 @@ def __getitem__(self, key):
         else:
             raise TypeError("key must be an index or slice")
 
-    def __array__(self, dtype=None):
-        return self._coords
+    def __array__(self, dtype=None, copy=None):
+        if copy is False:
+            raise ValueError("`copy=False` isn't supported. A copy is always created.")
+        elif copy is True:
+            return self._coords.copy()
```

Comment:
On thing I am still wondering: if we say that a copy is always created (in the error message above), then maybe we should not actually do the explicit copy here?

Of course, if one saved the `.coords` separately, one might still expect this actual copy ..

```
coords = geom.coords
coords_arr1 = np.asarray(coords, copy=True)
coords_arr2 = np.asarray(coords, copy=True)
```

i.e. one would expect in the case above that those two numpy arrays are not views of each other .. So I assume the above explicit copy is fine. We are just a bit inconsistent about the copy=False vs copy=True